newspapers.com.js

```diff
@@ -0,0 +1,104 @@
+{
+	"translatorID": "22dd8e35-02da-4968-b306-6efe0779a48d",
+	"label": "newspapers.com",
+	"creator": "Peter Binkley",
+	"target": "www.newspapers.com/clip/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2016-12-30 21:05:34"
+}
+
```

Comment:
We ask you to add a license here which you can simply copy from https://github.com/zuphilip/translators/wiki/Common-code-blocks-for-translators#licence-block (and change it with your name).